### ISSUE TITLE
[Accessibility] Labels do not bind to related controls

### ISSUE THREAD

**Issue by gazjoy** [Adyen/adyen-web#1693]
**Describe the bug**
When clicking a label on the drop-in payment solution the related control does not receive focus. [WCAG 2.1 - 1.3.1: Info and Relationships](https://www.w3.org/WAI/WCAG21/Understanding/info-and-relationships.html) says relationships need to be programatically determined or available in text. We use labels and form controls bound together to achieve this but it does not work the first time the form is encountered.

The Name on card field works ok but the others do not, I suspect this is because they have an <iframe> as their child. Once the field is clicked then the behaviour works but it should work by clicking the label, this can benefit users with an impaired fine motor control.

**To Reproduce**
1. Go to payment step
2. Use a mouse to click label "Card number"
3. Use mouse to click label "Expiry date"
4. Use mouse to click label "CVC/CVV"

**Expected behavior**
Clicking the <label> with a mouse should activate the related field, just as "Name on card" does.

**Screenshots**
![MMS-adyen-labels_AdobeExpress (1)](https://user-images.githubusercontent.com/628064/184666528-ebfe3953-40c3-4413-b2ce-28e6f7c6757a.gif)

**Desktop (please complete the following information):**
 - OS: macOS Monterey 12.5
 - Browser: Safari 15.6

**Additional context**
https://www.w3.org/WAI/WCAG21/Techniques/html/H44

**Comment by sponglord** [comment]
Hi @gazjoy - can you explain exactly what your Adyen-web implementation is i.e. what version, whether you are using Dropin or individual components? What exactly are we seeing in the screenshot you sent?
Because this functionality (click on Label, field gets focus) has been available since we launched the product

**Comment by gazjoy** [comment]
@sponglord
version is: "@adyen/adyen-web": "5.22.0"
implementation is: drop-in

**Comment by sponglord** [comment]
Hi @gazjoy - this is odd, and Safari specific behaviour... I'll take a look
